WordPress/Classes/ViewRelated/Post/PostListEditorPresenter.swift

```diff
@@ -15,11 +15,18 @@ protocol EditorAnalyticsProperties: AnyObject {
 struct PostListEditorPresenter {
 
     static func handle(post: Post, in postListViewController: EditorPresenterViewController, entryPoint: PostEditorEntryPoint = .unknown) {
-
-        // Return early if a post is still uploading when the editor's requested.
-        guard !PostCoordinator.shared.isUploading(post: post) else {
-            presentAlertForPostBeingUploaded()
-            return
+        if RemoteFeatureFlag.syncPublishing.enabled() {
+            // Return early if a post is still uploading when the editor's requested.
+            guard !PostCoordinator.shared.isUpdating(post) else {
```

Comment:
Not great naming – will probably rework later once we start tearing the old code out.